bin/make/test.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
```

Comment:
could we have this as makefile or is there an upside to using a bash script? if this was a makefile, we could include it from the main makefile with https://www.gnu.org/software/make/manual/html_node/Include.html

Reply:
handling whith those openers is quite confusing for me. bash or sh or something else, that is my lack of unix knowledge

Reply:
that line is correct, it makes it execute with bash, regardless of where bash is installed.

but if this file was instead a Makefile with makefile syntax instead of a bash script, we could include the shared makefile in the project makefile, instead of calling a bash script and have our own argument handling logic.